tests/src/test/scala/org/http4s/QValueSpec.scala

```diff
@@ -2,11 +2,13 @@ package org.http4s
 
 import cats.implicits._
 import cats.kernel.laws.discipline.OrderTests
+import org.http4s.testing.HttpCodecTests
 
 class QValueSpec extends Http4sSpec {
   import QValue._
 
   checkAll("QValue", OrderTests[QValue].order)
+  checkAll("HttpCodec", HttpCodecTests[QValue].httpCodec)
```

Comment:
I got this wrong myself, but I think `HttpCodec[QValue]` and `Order[QValue]` are the appropriate first arguments to this.  That's the instances that are being tested.  That's how most cats libraries use the laws.

Reply:
will change that